pulsar-client-cpp/lib/auth/athenz/ZTSClient.cc

```diff
@@ -143,7 +143,7 @@ std::string ZTSClient::ybase64Encode(const unsigned char *input, int length) {
 char *ZTSClient::base64Decode(const char *input) {
     BIO *bio, *b64;
     size_t length = strlen(input);
-    char *result = (char *)malloc(length);
+    char *result = (char *)calloc(length, sizeof(char));
```

Comment:
I have no idea how long `input` is and how many times this function will be called, but we should consider the overhead of initialization.

Also, in this particular case, `result[length] = '\0';` would look more purposeful because we want to return a null terminated string.